options/auto_retry_options.go

```diff
@@ -0,0 +1,18 @@
+package options
```

Comment:
I opted for a separate file to keep things readable. Nevermind the actual regexp strings - those we can tweak if the approach looks reasonable.